docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+# {{ ansible_managed }}
```

Comment:
What's this?

Reply:
I just remove it. It was a var for comment section which suppose to have version of the compose file, but somehow it was not populated. So, I just drop it.